portfolio/src/main/webapp/AboutMe.html

```diff
@@ -62,6 +62,17 @@ <h2>Vote for my next destination!</h2>
         </body>
       </html>
 
+      <form action="/destination-data" method="POST">
+      <select name="destination">
+        <option value="Brazil">Brazil</option>
+        <option value="Mexico">Mexico</option>
+        <option value="Alaska">Alaska</option>
+        <option value="Botswana">Botswana</option>
+        <option value="Ghana">Ghana</option>
+      </select>
```

Comment:
optional: If you want to allow users to enter custom destinations you could consider using the [HTML `<datalist>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist). E.g. 
```html
      <input list="destination-list" id="destination" name="destination" />
      <dalalist name="destination-list">
        <option value="Brazil">
        <option value="Mexico">
        <option value="Alaska">
        <option value="Botswana">
        <option value="Ghana">
      </datalist>
```

You might additionally consider populating the datalist options based on the returned destinations with votes